modules/bridge/src/main/ts/ephox/bridge/components/dialog/ColorInput.ts

```diff
@@ -1,17 +1,21 @@
-import { StructureSchema, ValueType } from '@ephox/boulder';
+import { FieldSchema, StructureSchema, ValueType } from '@ephox/boulder';
 import { Result } from '@ephox/katamari';
 
 import { FormComponentWithLabel, formComponentWithLabelFields, FormComponentWithLabelSpec } from './FormComponent';
 
 export interface ColorInputSpec extends FormComponentWithLabelSpec {
   type: 'colorinput';
+  swatchKey?: string;
```

Comment:
Does anyone know what `swatch` is here? It's a rather internal thing so I don't think this makes sense having thought about this more. I'd suggest just calling this either `key` or `storageKey`.

Reply:
I guess we do have `colorswatch` for the menuitem, but that doesn't make sense for `colorinput` though.